payments/db/sql_store.go

```diff
@@ -629,3 +629,108 @@ func (s *SQLStore) DeleteFailedAttempts(paymentHash lntypes.Hash) error {
 
 	return nil
 }
+
+// InitPayment initializes a payment.
+//
+// This is part of the DB interface.
+func (s *SQLStore) InitPayment(paymentHash lntypes.Hash,
```

Comment:
Given this is a completely new function, I'm expecting it be to written a way that has at least the same documentation level as the old one,
https://github.com/lightningnetwork/lnd/blob/995a1e75cd558ea5edc7731f5ba5c7441987b4bf/payments/db/kv_store.go#L189